jupyter_collaboration/handlers.py

```diff
@@ -184,13 +184,27 @@ async def open(self, room_id):
             try:
                 # Initialize the room
                 await self.room.initialize()
+            except web.HTTPError as e:
+                _, _, file_id = decode_file_path(self._room_id)
+                file = self._file_loaders[file_id]
+                # Close websocket and propagate error.
+                self.log.error(f"File {file.path} not found.\n{e!r}", exc_info=e)
```

Comment:
To avoid code duplication, I would only keep `except Exception as e` and test for `e` type to define the error message. This will allow to get only a single cleaning code avoiding discrepancy.

Are we also sure that any `web.HTTPError` implies the file is not found? This seems a bit to broad.

Reply:
> Are we also sure that any web.HTTPError implies the file is not found? This seems a bit to broad.

That's the error type the content manager raises for a file not found.